<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>TTAudioFileListWidget</class>
 <widget class="QWidget" name="TTAudioFileListWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>183</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>640</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="gbAudioFiles" >
     <property name="title" >
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <property name="spacing" >
       <number>4</number>
      </property>
      <item row="0" column="0" >
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="pbAudioFileOpen" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>0</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="audiofilelistwidget.qrc" >:/pixmaps/pixmaps/fileopen_24.xpm</iconset>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>4</number>
          </property>
          <item>
           <widget class="QPushButton" name="pbAudioEntryUp" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>25</width>
              <height>15</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset resource="audiofilelistwidget.qrc" >:/pixmaps/pixmaps/uparrow_18.xpm</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbAudioEntryDelete" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>25</width>
              <height>15</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset resource="audiofilelistwidget.qrc" >:/pixmaps/pixmaps/cancel_18.xpm</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbAudioEntryDown" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>25</width>
              <height>15</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset resource="audiofilelistwidget.qrc" >:/pixmaps/pixmaps/downarrow_18.xpm</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1" >
       <widget class="QTreeWidget" name="audioListView" >
        <property name="contextMenuPolicy" >
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="frameShape" >
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <property name="itemsExpandable" >
         <bool>false</bool>
        </property>
        <property name="columnCount" >
         <number>6</number>
        </property>
        <property name="sortingEnabled" >
         <bool>false</bool>
        </property>
        <column>
         <property name="text" >
          <string>Filname</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Length</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Version</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Bitrate</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Samplerate</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Delay</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="audiofilelistwidget.qrc" />
 </resources>
 <connections/>
</ui>
